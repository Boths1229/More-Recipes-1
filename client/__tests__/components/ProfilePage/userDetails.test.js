import React from 'react';
import { shallow } from 'enzyme';
import UserDetails from '../../../components/ProfilePage/userDetails';

let props;

const setup = () => {
  props = {
    username: 'sannitywo',
    email: 'sanni@test.com',
    myRecipesCount: 3,
    favRecipesCount: 4,
    createdAt: '2018-02-20T10:11:25.020Z',
  };

  return shallow(<UserDetails {...props} />);
};

describe('UserDetails Component', () => {
  it('should render correctly', () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('h5').length).toBe(5);
  });
});
