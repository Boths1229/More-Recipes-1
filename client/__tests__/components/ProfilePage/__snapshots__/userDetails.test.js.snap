// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserDetails Component should render correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-sm-12"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-12"
      >
        <h5>
          username: 
          <span>
            sannitywo
          </span>
        </h5>
      </div>
      <div
        className="col-sm-12"
      >
        <h5>
          Email: 
          <span>
            sanni@test.com
          </span>
        </h5>
      </div>
      <div
        className="col-sm-12"
      >
        <h5>
          Recipe Added: 
          <span>
            3
          </span>
        </h5>
      </div>
      <div
        className="col-sm-12"
      >
        <h5>
          Favorite Recipes: 
          <span>
            4
          </span>
        </h5>
      </div>
      <div
        className="col-sm-12"
      >
        <h5>
          Date Created: 
          <span>
            2018-02-20T10:11:25.020Z
          </span>
        </h5>
      </div>
    </div>
  </div>
</div>
`;
